lib/providers/compliance/mock.ts

```diff
@@ -0,0 +1,24 @@
+import { FillerComplianceConfiguration, FillerComplianceConfigurationProvider } from '.';
+
+export class MockFillerComplianceConfigurationProvider implements FillerComplianceConfigurationProvider {
+  constructor(private configs: FillerComplianceConfiguration[]) {}
+
+  async getConfigs(): Promise<FillerComplianceConfiguration[]> {
+    return this.configs;
+  }
+
+  async getAddrToEndpointsMap(): Promise<Map<string, Set<string>>> {
```

Comment:
maybe changing the name to be more clear whether this is endpoints to _include_ or endpoints to _exclude_ the address would be helpful for readability. Maybe `getExcludedEndpointsByAddr`